statistics/handle/bootstrap.go

```diff
@@ -266,18 +389,28 @@ func (h *Handle) initStatsBuckets(cache *statsCache) error {
 	}
 	req := rc[0].NewChunk()
 	iter := chunk.NewIterator4Chunk(req)
+	var lastTableID int64 = -1
 	for {
 		err := rc[0].Next(context.TODO(), req)
 		if err != nil {
 			return errors.Trace(err)
 		}
 		if req.NumRows() == 0 {
+			if limitSize == int64(req.NumRows()) && lastTableID != -1 {
```

Comment:
`==` or `<=`?

Reply:
`==` means the last table's data might incomplete.